vms/platformvm/service.go

```diff
@@ -868,7 +868,7 @@ func (s *Service) getPrimaryOrSubnetValidators(subnetID ids.ID, nodeIDs set.Set[
 				if err != nil {
 					return nil, err
 				}
-				isConnected := s.vm.uptimeManager.IsConnected(currentStaker.NodeID)
+				isConnected := s.vm.Network.Peers().Has(currentStaker.NodeID)
```

Comment:
I guess this is now race-y with the uptime information from the `uptimeManager`, but I don't know if we care here?

Reply:
It's indeed racey and more than being racey it's not reflect the same outcome with `IsConnected` as a peer could be benched ([here](https://github.com/ava-labs/avalanchego/blob/joshua-kim/remove-is-connected/snow/networking/router/chain_router.go#L568))